src/main/plugin/iso19139.ca.HNAP/templates/hnap_nonspatial-english.xml

```diff
@@ -422,6 +422,9 @@
          <gmd:language>
             <gco:CharacterString>eng; CAN</gco:CharacterString>
          </gmd:language>
+        <gmd:language>
```

Comment:
Check to remove these changes in the templates.

Reply:
I talked to User again and confirmed that they mostly wanted the "+" option at add new languages.  They are find with the templates not changings as they are more similar to FGP templates.

@wangf1122 - please removed the changed on the templates.

Reply:
Removed for all these template. Please check